### PR TITLE
AUTH-482: set required-scc for openshift workloads

### PR DESCRIPTION
**Pull request by liouk**

### CODE REVIEW

**Comment by liouk** [comment]
/hold
need to verify SCC

**Review by jhadvig — CHANGES_REQUESTED** [review]
@liouk could you please update the unit tests accordingly ?

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Comment by jhadvig** [comment]
QE Approver:
/assign @yapei 
Docs Approver:
/assign @opayne1 
PX Approver:
/assign @RickJWagner

File: bindata/assets/deployments/console-deployment.yaml
Comment (yapei) [inline bindata/assets/deployments/console-deployment.yaml:24]: sorry @liouk  May I know why it is pinned to `restricted-v2` instead of other default SCCs?

Comment (liouk) [reply bindata/assets/deployments/console-deployment.yaml:24]: The console pods are anyway getting admitted with `restricted-v2`, as they don't need more privileged access. Here we are just pinning the default SCC that the pods are already getting admitted with.

**Comment by opayne1** [comment]
Verified with @bergerhoffer 

/label docs-approved

**Pull request merged**
